### PR TITLE
refactor(@vtmn/css): remove useless props, align icon visually, set fix height

### PR DESCRIPTION
**Pull request by GaspardMathon**
## Pull Request checklist

🚨 Please review the [guidelines for contributing](../blob/main/CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch**. Don't request your main!
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.
- [x] Check PR name, it must follow the following pattern : TYPE(@vtmn/PACKAGENAME): SHORT DESCRIPTION

## Description

- Remove useless props and props already set by the postcss processor.
- Icon with text are now visually centered aligned
- The height of the different button sizes is fixed

## Does this introduce a breaking change?

- No

❤️ Thank you!

### CODE REVIEW

File: packages/sources/css/src/components/button/src/index.css [hunk 1]
```diff
@@ -41,7 +31,7 @@
 
 .vtmn-btn:disabled {
   cursor: not-allowed;
-  opacity: var(--vtmn-state_disabled_opacity);
+  opacity: 0.38;
```
Comment (lauthieb) [inline packages/sources/css/src/components/button/src/index.css:34]: Why?

Comment (lauthieb) [reply packages/sources/css/src/components/button/src/index.css:34]:
```suggestion
  opacity: var(--vtmn-state_disabled_opacity);
```

File: packages/sources/css/src/components/button/src/index.css [hunk 2]
```diff
@@ -211,6 +200,7 @@
 
 .vtmn-btn--icon-alone span[class^='vtmx-'] {
   font-size: var(--vtmn-typo_text-2-font-size);
+  transform: translateY(rem(1px));
```
Comment (lauthieb) [inline packages/sources/css/src/components/button/src/index.css:203]: It feel nicer like this (IMHO)
```suggestion
  transform: translateY(rem(2px));
```

File: packages/sources/css/src/components/button/src/index.css [hunk 3]
```diff
@@ -2,32 +2,22 @@
...
 @import '@vtmn/css-design-tokens/src/typography';
-@import '@vtmn/css-design-tokens/src/opacities';
 
 /* Default button */
 .vtmn-btn {
   box-shadow: inset 0 0 0 rem(2px) transparent;
-  font-size: var(--vtmn-typo_text-1-font-size);
-  min-height: rem(48px);
+  font-size: var(--vtmn-typo_text-2-font-size);
+  height: rem(48px);
```
Comment (lauthieb) [inline packages/sources/css/src/components/button/src/index.css:10]: Your proposition arrives with a new bug:
Before:
image


After:
image

**Review by lauthieb — CHANGES_REQUESTED** [review]

File: packages/sources/css/src/components/button/src/index.css (hunk 3 above)
Comment (GaspardMathon) [reply packages/sources/css/src/components/button/src/index.css:11]: corrected

**Pull request closed without merging**